scss/_nav.scss

```diff
@@ -46,6 +48,8 @@
 
     &.disabled {
       color: $nav-link-disabled-color;
+      pointer-events: none;
+      cursor: default;
```

Comment:
These two lines are unnecessary if the base `.nav-link` already gets it from L25-26.

Reply:
Done!